contracts/Remittance.sol

```diff
@@ -15,52 +15,53 @@ contract Remittance is Stoppable {
     uint constant TX_FEES = 2000;
     uint constant CANCELLATION_DELAY = 5 days;
     
-    mapping(bytes32 => Transaction) private transactions;
-    uint private fees;
+    mapping(bytes32 => Order) private transactions;
+    mapping(address => uint) private fees;
```

Comment:
Are you intentionally making it difficult for people to know the status of the contract?

Reply:
I intentionally wanted to make these variables private to ensure confidentiality

Reply:
Do you know that all data is public? There is no confidentiality. All you are doing is asking people to do a sha3 and a `web3.eth.getStorageAt` to get the data.
The only people who will manage that are potential attackers.
Best practice: make all data public.

Reply:
Gotcha this time. Quite difficult to admit that everything is public and tracable!